### PR TITLE
Exposes job.PriorityClass.value as modifiable value

### PR DESCRIPTION
**Pull request by mapk-amazon**
Addresses this issue. 
https://github.com/galaxyproject/galaxy-helm/issues/450

### CODE REVIEW

File: galaxy/templates/priorityclass-job.yaml [hunk 1]
```diff
@@ -5,8 +5,12 @@ metadata:
   name: {{ include "galaxy.pod-priority-class" . }}
   labels:
     {{- include "galaxy.labels" . | nindent 4 }}
-value: -1000
+value: {{ if .Values.jobs.priorityClass.value -}} 
+       {{ .Values.jobs.priorityClass.value}} 
+       {{- else -}} 
+       -1000 
+       {{- end}}
```
Comment (nuwang) [inline galaxy/templates/priorityclass-job.yaml:12]:
```suggestion
value: {{ default "-1000"  .Values.jobs.priorityClass.value -}} 
```

File: galaxy/templates/priorityclass-job.yaml [hunk 2]
```diff
@@ -5,8 +5,12 @@ metadata:
...
 preemptionPolicy: Never
 globalDefault: false
-description: "Galaxy jobs will run when cluster resources are available and will not preempt existing workloads."
+description: "By default, Galaxy jobs will not preempt existing workloads but will run when cluster resources are available. Depending on the cluster setup, the preemption value can be modified to prompt scaling of the cluster for the computation."
```
Comment (nuwang) [inline galaxy/templates/priorityclass-job.yaml:15]: I think some guidance on what values would elicit the desired behaviour in the autoscaler case would also be helpful.

File: galaxy/templates/priorityclass-job.yaml [hunk 3]
```diff
@@ -5,8 +5,8 @@ metadata:
   name: {{ include "galaxy.pod-priority-class" . }}
   labels:
     {{- include "galaxy.labels" . | nindent 4 }}
-value: -1000
+value: {{ default "-1000"  .Values.jobs.priorityClass.value -}}
```
Comment (nuwang) [inline galaxy/templates/priorityclass-job.yaml:8]: This needs a bit of testing, but the last line chomp might break things?
```suggestion
value: {{ default "-1000"  .Values.jobs.priorityClass.value }} 
```

Comment (mapk-amazon) [reply galaxy/templates/priorityclass-job.yaml:8]: Yeah, the next line is pulled up to line 8 now. Removing the Minus/Dash.

File: galaxy/templates/priorityclass-job.yaml (hunk 2 above)
Comment (mapk-amazon) [reply galaxy/templates/priorityclass-job.yaml:11]: Thank you, sounds great!

**Review by nuwang — APPROVED** [review]

**Comment by nuwang** [comment]
@mapk-amazon Thanks for this! Looks good.

**Pull request merged**